ts/navigation/MainNavigator.tsx

```diff
@@ -56,9 +56,7 @@ const styles = StyleSheet.create({
   labelStyle: {
     ...makeFontStyleObject(Platform.select),
     marginBottom: 7,
```

Comment:
try to limit the usage of margins to position elements in their own container. if you want to pad the container, add padding to the container. using margins to position elements makes them not reusable.